Server/VikingVault.Services/ScopedProcessingService.cs

```diff
@@ -68,8 +68,9 @@ private void AddPayingUserAutomaticTransaction(AutomaticPayment automaticPayment
                 Type = "Payment",
                 Date = currentDateTime,
                 Currency = "Ron",
-                Amount = automaticPayment.Amount,
-                OtherParty = automaticPayment.ReceivingCompany.FirstName
+                Amount = -automaticPayment.Amount,
```

Comment:
The name of this service is confusing. Why Scoped, Why Processing? Also, this - added here and there confuse me since it is hard for me to understand why we`re doing it. If you leave the project and I`m going to continue it, things are going to be hard for me to understand.

Reply:
1. ScopedProcessing because that is what it looks like on Microsoft docs.
2. When we create the transaction we add a negative amount because the user is basically paying which mean that his balance changed with -amount

Reply:
1. Still, it`s confusing, since I cannot see the link between what it does and the name.
2. That should be formalized somehow, because it is prone to errors.